### PR TITLE
[9.0][ADD] pos_payment_restriction

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: pos_payment_restriction/static/src/js/models.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+*/
+
+odoo.define('pos_payment_restriction.models', function (require) {
+    "use strict";
+
+    var models = require('point_of_sale.models');
+
+    var _paymentlineproto = models.Paymentline.prototype;
+    models.Paymentline = models.Paymentline.extend({
+    });
```
Comment (chienandalu) [inline pos_payment_restriction/static/src/js/models.js]: I guess this is WIP, isn't it? :sweat_smile:

Comment (rousseldenis) [reply pos_payment_restriction/static/src/js/models.js]: Done

**Comment by legalsylvain** [comment]
Hi @rousseldenis. thanks for sharing this module. 
I didn't made a deep review, but in the first sight, I'd better see the settings, in the journal model, to allow to disable / enable both feature payment method by payment method.
What do you think ?

**Comment by rousseldenis** [comment]
> Hi @rousseldenis. thanks for sharing this module.
> I didn't made a deep review, but in the first sight, I'd better see the settings, in the journal model, to allow to disable / enable both feature payment method by payment method.
> What do you think ?

It's quite linked to POS behaviour, so IMHO it should stay there.

**Comment by legalsylvain** [comment]
> It's quite linked to POS behaviour, so IMHO it should stay there.

answered here : https://github.com/OCA/pos/pull/598#issuecomment-773224298

**Pull request closed without merging**